<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>598</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btn_gameexit">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>540</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_title_npvp">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Network Game Player1 AI vs Player2 AI</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_dip">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_dport">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>50</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="lb_dip">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Server IP</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_dport">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Server PORT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_connect">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>40</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_p1">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>140</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>You(P1)</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_p2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>200</y>
     <width>131</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Opponent(P2)</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_score">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>300</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>SCORE</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcd_score_b">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>330</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QLCDNumber{
    color: rgb(255, 255, 255);    
    background-color: rgb(0, 0, 0);
}</string>
   </property>
   <property name="smallDecimalPoint">
    <bool>false</bool>
   </property>
   <property name="digitCount">
    <number>5</number>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Filled</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcd_score_w">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>330</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLCDNumber{
    color: rgb(0, 0, 0);   
    background-color: rgb(250, 250, 250);
}</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
